solidity/test/BancorConverter.js

```diff
@@ -593,13 +593,13 @@ contract('BancorConverter', accounts => {
     it('should throw when attempting to get the purchase return while the converter is not active', async () => {
         let converter = await initConverter(accounts, false);
 
-        await utils.catchRevert(converter.getPurchaseReturn.call(reserveToken.address, 500));
+        await utils.catchRevert(converter.getPurchaseReturn.call(reserveToken.address, 500, 0));
```

Comment:
see the converter - these should just be private

Reply:
@yudilevi : See the comment above.